J/JSBSim/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+using BinaryBuilder
+
+name = "JSBSim"
+version = v"1.1.6"
+julia_version = v"1.5.3"
+
+# Collection of sources required to build JSBSim
+sources = [
+    GitSource("https://github.com/bcoconni/jsbsim.git",
+              "f3911ba6354f271d7ecaeb0078d7bc7fe8eccb65"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jsbsim
+mkdir build && cd build
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+    FLAGS+=(-DCMAKE_CXX_FLAGS_RELEASE="-D_POSIX_C_SOURCE")
+fi
+export PATH=$PATH:$prefix
```

Comment:
In `${prefix}` there are no executables, `${bindir}` [is already](https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/de1e65ff11459ae0ed8b8d1318ceea1c0f7c2893/src/Runner.jl#L887) in `PATH`, anyway we _don't_ want to run Julia, the version number should be read from the header files

Reply:
> anyway we *don't* want to run Julia

Why not ?

Reply:
Because we can't :slightly_smiling_face: This is a cross-compilation environment, we can't run foreign executables built for the target (apart from 2-3 platforms)

Reply:
Anyway the whole point of trying to access Julia executable is to get prefix path of `CxxWrap`:
```julia
using CxxWrap
print(CxxWrap.prefix_path())
```
If we can access this information (in a portable way) without executing the script above with `julia` then my problem is solved.

Reply:
> This is a cross-compilation environment, we can't run foreign executables built for the target (apart from 2-3 platforms)

Ha OK ! Silly me :roll_eyes:

Reply:
@fingolfin is the master of CxxWrap-related packages, that's why I'm trying to summoning help for help :slightly_smiling_face:

Reply:
Sorry still at dinner with the kids 🍝🥄

Reply:
No rush!

Reply:
I have removed the `export PATH=...` thing.